code/modules/mob/living/carbon/human/human.dm

```diff
@@ -215,6 +215,13 @@
 	if(istype(MB))
 		MB.RunOver(src)
 
+	if(ishuman(AM) && lying)

```

Comment:
This should not apply if the player is:
- walking.
- is missing legs.
- flying or floating.
- in zero gravity.
- a dwarf.
- being thrown.
- on a vehicle (in which case MORE damage should occur!).